### PR TITLE
CHANGE (CodeAnalyzer): @W-14203641@: 3.17.0 Release Summary work

### PR DESCRIPTION
**Pull request by teresa-allen-sfdc**
Promote PerformNullCheckOnSoqlVariables from pilot to GA
Release summary work
Fix intro (per below)

Here are the new and changed features in recent updates of Salesforce Code Analyzer (Code Analyzer) and the Salesforce Code Analyzer (beta) Visual Studio (VS) Code extension.

We publish the latest Code Analyzer monthly.

Run sfdx plugins to display the version of Code Analyzer installed on your computer.
Run sfdx plugins:update to update Code Analyzer and help to the latest version.

### CODE REVIEW

File: docs/_articles/en/v3.x/release-information.md [hunk 1]
```diff
@@ -4,14 +4,25 @@ lang: en
...
 * Follow these [instructions](./en/v3.x/getting-started/install/#upgrade-plug-in) to update Code Analyzer
 
+## [v3.17.0](https://github.com/forcedotcom/sfdx-scanner/tree/v3.17.0) (10-03-2023)
+
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v3.17.0...v3.16.0)
+
+### Release Summary
+
+* NEW (CodeAnalyzer): We made some updates to the RetireJS vulnerability database.
+* NEW (GraphEngine): One recently released Graph Engine pilot rule is now generally available: PerformNullCheckOnSoqlVariables.
```
[the remark this answers is not in the document]
Comment (teresa-allen-sfdc) [reply docs/_articles/en/v3.x/release-information.md:22]: @jfeingold35 
1. Do we need an entry about that? I'm not sure. We really just dropped VS Code Extension from the name. A question for John then. 
2. Huh, this one didn't make it onto the G doc draft. I'll have to add it.

File: docs/_articles/en/v3.x/salesforce-graph-engine/rules.md [hunk 2]
```diff
@@ -50,14 +49,12 @@ To run all Graph Engine rules and all pilot rules, run: ```sfdx scanner:run:dfa
 
 ```sfdx scanner:run:dfa --with-pilot --engine sfge --projectdir /project/dir --target /project/dir/target1```
 
-<br>
```
[the remark this answers is not in the document]
Comment (teresa-allen-sfdc) [reply docs/_articles/en/v3.x/salesforce-graph-engine/rules.md:53]: The <br> makes things look STRANGE in preview though

[the remark this answers is not in the document]
Comment (teresa-allen-sfdc) [reply docs/_articles/en/v3.x/salesforce-graph-engine/rules.md:53]: Done

File: docs/_articles/en/v3.x/release-information.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (teresa-allen-sfdc) [reply docs/_articles/en/v3.x/release-information.md:22]: I added the CHANGE, but not the rename. Would like John's opinion on that.

**Review by jfeingold35 — APPROVED** [review]

**Pull request merged**